pkg/minishift/addon/addon_meta.go

```diff
@@ -89,6 +92,13 @@ func (meta *DefaultAddOnMeta) GetValue(key string) string {
 	return meta.headers[key].(string)
 }
 
+func (meta *DefaultAddOnMeta) OpenShiftVersion() string {
```

Comment:
Could return a struct instead of a single string. The content of the struct could be the version together with a range